src/bindings/js/node/tests/basic.test.js

```diff
@@ -0,0 +1,96 @@
+const ov = require('../build/Release/ov_node_addon.node');
+const path = require('path');
+
+function getModelPath(is_fp16=false){
```

Comment:
I propose to use camelCase style instead of python_case
```suggestion
function getModelPath(isFP16=false){
```

Reply:
Ok